ignite/chainconfig/chainconfig.go

```diff
@@ -0,0 +1,30 @@
+package chainconfig
+
+import (
+	"github.com/ignite/cli/ignite/chainconfig/config"
+	v0 "github.com/ignite/cli/ignite/chainconfig/v0"
+	v1 "github.com/ignite/cli/ignite/chainconfig/v1"
+	"github.com/ignite/cli/ignite/pkg/xfilepath"
+)
+
+var (
+	// ConfigDirPath returns the path of configuration directory of Ignite.
+	ConfigDirPath = xfilepath.JoinFromHome(xfilepath.Path(".ignite"))
+
+	// ConfigFileNames is a list of recognized names as for Ignite's config file.
+	ConfigFileNames = []string{"config.yml", "config.yaml"}
+
+	// LatestVersion defines the latest version of the config.
+	LatestVersion config.Version = 1
+
+	// Versions holds config types for the supported versions.
+	Versions = map[config.Version]config.Converter{
+		0: &v0.Config{},
+		1: &v1.Config{},
+	}
+)
+
+// DefaultConfig returns a config for the latest version initialized with default values.
+func DefaultConfig() *v1.Config {
+	return v1.DefaultConfig()
+}
```

Comment:
```suggestion
// Current config definition
type Config = v1.Config

// DefaultConfig returns a config for the latest version initialized with default values.
func DefaultConfig() *Config {
	return v1.DefaultConfig()
}
```

A type alias can be interesting because then you no longer need to replace all `chainconfig.Config` by the current config `v1.Config`. This decreases the changes of this PR and also facilitates the introduction of new configs in the futures.

Thanks to this type alias, when a new config is needed, you only need to change the type alias and the `DefaultConfig` function. All the other packages refers to `chainconfig.Config` and never refers to `chainconfig/vX`.

Reply:
Changed in 876ed832e900acea6aa479fa3b6090aadddd6efe